CMakeLists.txt

```diff
@@ -383,9 +384,9 @@ if(SANITIZERS)
     message(FATAL_ERROR "Linker did not accept requested flags, you are missing required libraries.")
   endif()
 endif()
-target_link_options(sanitize_interface INTERFACE ${SANITIZER_LDFLAGS})
```

Comment:
In commit 9e7eb06a24b670e8e72e0233a63bbc67733d4adb
Nit: Is this change related to the actual change introducing `FUZZ_LIBS` in this commit?

Reply:
Unrelated changes have been dropped.